avmp/utils/logging_utils.py

```diff
@@ -5,11 +5,11 @@
 
 import os
 import sys
-import json
 import logging
+import json
```

Comment:
Why is this change needed?

Reply:
It isn't I had to make another commit to redo the github workflow